make rpm fails on Red Hat Enterprise Linux 8.3

`/usr/bin/pylint-3 --rcfile .pylintrc pglookout/ test/
************* Module test.test_lookout
test/test_lookout.py:643:24: W0612: Unused variable 'standby_nodes' (unused-variable)

------------------------------------------------------------------
Your code has been rated at 9.99/10 (previous run: 9.99/10, +0.00)

make[1]: *** [Makefile:24: pylint] Error 4
make[1]: Leaving directory '/home/portalify/pglookout/rpm/BUILD/pglookout'
virhe: Bad exit status from /var/tmp/rpm-tmp.59pw2K (%check)

RPM käännösvirheitä:
    Bad exit status from /var/tmp/rpm-tmp.59pw2K (%check)
make: *** [Makefile:42: rpm] Virhe 1
`

--- a/test/test_lookout.py
+++ b/test/test_lookout.py
@@ -640,7 +640,7 @@ def test_node_map_when_only_observer_sees_master(pgl):
             "fetch_time": "2014-08-28T14:26:51.069891Z",
         }
     }
-    master_instance, _, standby_nodes = pgl.create_node_map(cluster_state, observer_state)
+    master_instance, _, _ = pgl.create_node_map(cluster_state, observer_state)
     assert master_instance == "10.255.255.10"
     # because observer saw it and its fetch time is later than cluster time
     assert master_instance in pgl.connected_master_nodes

